router-tests/go.mod

```diff
@@ -176,5 +176,5 @@ require (
 replace (
 	github.com/wundergraph/cosmo/demo => ../demo
 	github.com/wundergraph/cosmo/router => ../router
-// github.com/wundergraph/graphql-go-tools/v2 => ../../graphql-go-tools/v2
+//  github.com/wundergraph/graphql-go-tools/v2 => ../../graphql-go-tools/v2
```

Comment:
I don't think you meant to commit this

Reply:
Why not? I corrected the identation.

Reply:
I see. But the other lines are a tab character, and this is a second space. It still doesn't align. At least not on my end.